mathesar_ui/src/header/Header.svelte

```diff
@@ -15,17 +18,27 @@
   import {
     Icon,
     Button,
+    Dropdown,
   } from '@mathesar-components';
 
   import SchemaSelector from './schema-selector/SchemaSelector.svelte';
   import ImportIndicator from './import-indicator/ImportIndicator.svelte';
 
-  function createNewTable() {
+  async function createEmptyTable() {
+    const response = await postAPI<{ id: number, name: string }>('/tables/', {
+      schema: $currentSchemaId as number,
```

Comment:
The linter error for `Unsafe assignment of an any value`, is a bug/known limitation in the eslint plugin for svelte, when dealing with stores. Reference: https://github.com/sveltejs/eslint-plugin-svelte3/issues/104

This error is not thrown if we use the get store method i.e., get(currentSchemaId).

This issue only occurs when dealing with .svelte files. When this logic is moved to the store, the assertion will not be required.